src/main/java/duke/internal/Storage.java

```diff
@@ -0,0 +1,56 @@
+package duke.internal;
+
+import java.io.IOException;
```

Comment:
similarly, maybe auto ordering can be used here for consistent import statements